homeassistant/components/script/__init__.py

```
@@ -335,6 +339,16 @@ async def async_turn_on(self, **kwargs):
         self.hass.async_create_task(coro)
```

The result of this is never checked, and exceptions may be silently swallowed.
Should that be fixed?

---

Yes

---

How?